dev/io.openliberty.checkpoint_fat/publish/servers/checkpointFaces-backendServices/jvm.options

```diff
@@ -0,0 +1 @@
+-Dcom.ibm.ws.beta.edition=true
```

Comment:
This doesn't matter, but I'm guessing the backendServices doesn't need beta functionality.

Reply:
Agree.  May as well remove it now.